codespell_lib/data/dictionary.txt

```diff
@@ -443,6 +451,7 @@ abundunt->abundant
 aburpt->abrupt
 aburptly->abruptly
 abuseres->abusers
+abusin->abusing
```

Comment:
Could be `a basin` or `a buy-in`, but that's probably far-fetched. Let's disregard such suggestions for now.